packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -30,7 +30,16 @@ export const makeZoeSeatAdminKit = (
 ) => {
   const payoutPromiseKit = makePromiseKit();
   const offerResultPromiseKit = promises.offerResult || makePromiseKit();
-  const exitObjPromiseKit = promises.exitObj || makePromiseKit();
+  const exitObjPromiseKitPromise =
+    promises.exitObj ||
+    Promise.resolve(
+      // Offerless seat case
+      harden({
+        exit: () => {
+          throw new Error(`Offerless seats may not be exited`);
+        },
+      }),
+    );
```

Comment:
```suggestion
  const exitObjP = promises.exitObj ? promises.exitObj.promise :
      // Offerless seat case
      harden({
        exit: () => {
          throw new Error(`Offerless seats may not be exited`);
        },
      });
```

Reply:
I renamed the long variable name because it need not be a promise, and it need not have anything to do with a promise-kit. It is just an ERef to an exitObj.